composer.json

```diff
@@ -15,7 +15,7 @@
         "doctrine/doctrine-bundle": "^1",
         "sonata-project/doctrine-orm-admin-bundle": "^3",
         "zicht/framework-extra-bundle": "^8.0",
-        "sensio/framework-extra-bundle": "^3.0"
+        "sensio/framework-extra-bundle": "^5"
```

Comment:
Dit is niet echt backwards compatible, dan moet dit echt even naar `release/6.x`.  En fix nog even een changelog